cmd/up/up.go

```diff
@@ -109,7 +109,7 @@ func Up() *cobra.Command {
 				return err
 			}
 
-			if err := loadDevOverrides(dev, namespace, k8sContext, forcePull, remote); err != nil {
+			if err := loadDevOverrides(dev, namespace, k8sContext, forcePull, remote, autoDeploy); err != nil {
```

Comment:
There is an option (`MarkHidden`) in cobra to not show an arg in the help command. Let add the hidden option to the `autodeploy` flag and also remove `autodeploy` from the docs, but keeping the flag hidden for now